recipe/meta.yaml

```diff
@@ -1,14 +1,15 @@
 {% set name = "pluggy" %}
-{% set version = "0.4.0" %}
-{% set sha256 = "dd841b5d290b252cf645f75f3bd37ceecfa0f36394ab313e4f785fe68a4081a4" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "159cc783e056c07da6552aa5aef6b1e6c0064b4f18bd49c531fd2d40aafb0ea3" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.zip
-  url: https://pypi.io/packages/source/{{ name [0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  # url: https://pypi.io/packages/source/{{ name [0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  url: https://pypi.python.org/packages/1b/a9/6f5f80b75a8d84d21a8a13486fe26a2da9f043f93b464b2e3928be256dc4/pluggy-0.3.1.tar.gz#md5=ecdd791e309f60668b66fec97c2ee7db
```

Comment:
Would just try changing `https://pypi.io/` to `https://pypi.python.org/` and otherwise try using this URL unchanged.

Reply:
`pypi.io` url works with `.tar.gz`. You shouldn't use `pypi.python.org` urls

Reply:
So this is a very old version that the user is trying to get. Have often found in cases like this it is simply not available using the `pypi.io` URL, which is unfortunate. Hence the suggestion to use the legacy URL.

Reply:
https://pypi.io/packages/source/p/pluggy/pluggy-0.3.1.tar.gz works, but https://pypi.io/packages/source/p/pluggy/pluggy-0.3.1.zip doesn't

Reply:
That url didn't work.  Also when I used that url in my browser, it didn't download the file.

Reply:
@sannykr, which url are you talking about? There are two I mentioned at https://github.com/conda-forge/pluggy-feedstock/pull/2#discussion_r137888623

Reply:
@isuruf : Thanks. got that. I just need to add tar.gz extension and use pypi.io in the url then it will work. It seems the meta.yaml file is old, and it has used .zip extension that I didn't notice.